tests/system/fim/common_tasks/transfer_output_files.yml

```diff
@@ -0,0 +1,37 @@
+---
+- block:
+  - name: Copy the output files from Agents to Ansible Controller host | Linux
+    fetch:
+      src: "{{ agents_linux_output_path }}/{{ files_generated_output_path }}"
+      dest: "agents_outputs/added/{{ files_generated_output_path }}-{{ inventory_hostname }}"
+      flat: yes
+    when:
+     - inventory_hostname in groups['linuxagents']
+
+  - name: Copy the output files from Ansible Controller to the Managers' hosts | Linux
+    synchronize:
+      src: "agents_outputs/added/"
+      dest: "{{ agents_results_manager_path }}/added/"
+    when:
+      - inventory_hostname in groups['masters']
+  when:
+    - inventory_hostname in groups['linuxagents'] or inventory_hostname in groups['masters']
+  become: yes
+
+- block:
+  - name: Copy the output files from Agents to Ansible Controller host | Windows
+    fetch:
+      src: "{{ agents_windows_output_path }}/{{ files_generated_output_path }}"
+      dest: "agents_outputs/{{ event }}/{{ files_generated_output_path }}-{{ inventory_hostname }}"
+      flat: yes
+    when:
+      - inventory_hostname in groups['windowsagents']
+
+  - name: Copy the output files from Ansible Controller to the Managers' hosts | Windows
+    synchronize:
+      src: "agents_outputs/added/"
+      dest: "{{ agents_results_manager_path }}/added/"
+    when:
+      - inventory_hostname in groups['masters']
+  when:
+    - inventory_hostname in groups['windowsagents'] or inventory_hostname in groups['masters']
```

Comment:
I see that these tasks are not imported in any part of the default scenario. Also, the tasks included here are already included in each action Ansible tasks file.

Maybe this file should be removed.

kr,